scripts/uart_terminal/README.rst

```diff
@@ -0,0 +1,66 @@
+.. _uart_terminal_script:
+
+UART terminal script
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The UART terminal script can be used with any setup that uses several development kits to start a terminal connection to all DKs at the same time.
+This can save time needed to manually establish multiple connections, which speeds up :ref:`gs_testing`.
+
+For example, when working with the :ref:`nrf53_audio_app` in the BIS mode with several receivers, you can use the script instead of manually opening a terminal connection to each DK.
+
+Overview
+********
+
+Once started, the script checks for existing serial connections by running the ``nrfjprog --com`` command in the background.
+It then starts a terminal connection for each DK's COM port with the number corresponding to the one specified in the :file:`get_serial_ports.py` file.
+By default, the terminal connection is opened for the virtual COM port (VCOM) of the application core (``VCOM0``).
+
+Requirements
+************
+
+The script source files are located in the :file:`scripts/uart_terminal` directory.
+
+To use the script, make sure that you have the following software installed for your operating system:
+
+* Windows: Python and PuTTy
+* Linux: Python, Terminator, and Minicom
```

Comment:
we are missing macOS here.

also, if working with NCS requires special software installed, then such software should be part of the NCS Toolchain manager.

**cc** @martingalvan-nordic

Reply:
That is a valid point. Please take into account that using this tool is not a requirement, but that may not make any difference.

Reply:
Hi all. If the intention is to add these to the NCS toolchains we bundle and distribute using the toolchain-manager, it would be good to discuss this with @ihansse and @kenr first.

Reply:
I am not convinced this is the right way to go. This is an optional script (user may have a preferred terminal)  and the other scripts in nrf/scripts have to install their own requirements/tools. I believe this should be left as is and not additional tools added to NCS toolchains.

Reply:
I disagree here, you provide a script which requires additional tools in order to run, then we should ensure such tools are available.

> This is an optional script 

That you can say about many things, but if we follow this approach then users can easily end-up having to figure out which additional n-tools they need to install in order to work efficiently with NCS.

Reply:
For the other scripts in the nrf/scripts are the requirements scripts run when the NCS tools are installed so as to install the tools these require?

Reply:
yes, so in case a python script requires a specific pypi package, then that information is added to `scripts/requirements-<area>.txt`.

If a dedicated tool is needed, then it's added to `scripts/tools-versions-*.txt`.

That way it both gets documented which tools are used in NCS, as well as serving as input when the NCS Toolchain is built.

Reply:
Furthermore those files also serves as information to users who want to manually install tools they need.

Reply:
Thanks, so if I add a requirements file this would resolve the issue?

Reply:
@gWacey as per my previous comment, if the intention is to add new software to the NCS toolchains we bundle and distribute using the toolchain-manager, it would be good to discuss this with @ihansse and @kenr first. pypi packages should ideally work out-of-the box, but other tools (i.e. what goes in tools-versions-*.txt) requires the Wayland systems team to actually implement the downloading/bundling of the new tools into the toolchains.

Reply:
Apologies I am new to this, I will arrange a discussion on this with those suggested, should anyone else be involved?

Reply:
Might be worth to keep myself and @thst-nordic in the loop to see what's the result of that discussion.

Reply:
> Thanks, so if I add a requirements file this would resolve the issue?

@gWacey for the tools in question you should update the existing `scripts/tools-<OS>.txt` file with the needed info.
@martingalvan-nordic can help / ensure the correct path for the bundling into the toolchain manager, but I believe adding the tools to the right `scripts/tools-<OS>.txt` should help moving this PR forward, then @martingalvan-nordic can ensure they correctly gets downloaded and bundled.

> Apologies I am new to this,

No need for apologies, we have all been new at some point.
We are here to help you on this and explaining, while at the same trying to ensure some consistency in our SDK 😉

Reply:
Agreed, but please do talk to @kenr about this before moving forward. It would be good that we were all on the same page regarding the timeline for introducing this, etc.

Reply:
Thanks all